.env.example

```
@@ -0,0 +1,14 @@
+
+# Development mode (enables debug features)
+# ARCO_DEVELOPMENT=true
+
+# Start page override (useful for development)
+# ARCO_START_PAGE=/repositories
+
+# Custom Arco Cloud RPC URL (defaults to production)
+# ARCO_CLOUD_RPC_URL=https://api.arco.cloud
+
+# Use file-based keyring instead of system keyring
+# ARCO_USE_FILE_KEYRING=true
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)